pandas/io/packers.py

```diff
@@ -251,7 +251,6 @@ def dtype_for(t):
             'complex128': np.float64,
             'complex64': np.float32}
 
-# numpy 1.6.1 compat
 if hasattr(np, 'float128'):
     c2f_dict['complex256'] = np.float128
```

Comment:
This is not just numpy compat (hence the removed command) - including `'complex256': np.float128` into `c2f_dict` does not work on my windows machine, for example

Reply:
maybe a comment to that effect?  "windows compat"or "windows 32 bit compat" or "[whatever would be accurate] compat"?

Reply:
ok